start.sh

```diff
@@ -3,6 +3,8 @@
 # Chown volumes so that nginx can read/write
 chown www-data:www-data /usr/share/nginx/html/images
 chown www-data:www-data /usr/share/nginx/html/LocalSettings.php
+chown -R www-data:www-data /tmp/extensions/
+cp -r /tmp/extensions/* /usr/share/nginx/html/extensions/
```

Comment:
Great feedback :+1: :smile: . Understood the intention of this image, will close this pull request.

One more thing, hoping for your suggestion :smile: . In regard to ephemeral guideline, how does "LocalSettings.php" differ to "extensions\" folder? Why not write Dockerfile to ``ADD`` it as well?